### PR TITLE
Lead member

### PR DESCRIPTION
**Pull request by maximenkom**
added rest controller with userDTO params on "/submitUserForm" mapping, adds user into DB and sends mail to user with his login and password

### CODE REVIEW

File: src/main/java/ua/cv/tim/controller/UserController.java [hunk 1]
```diff
@@ -8,16 +10,71 @@
...
+	@RequestMapping(value = "/add", method = RequestMethod.GET)
+	public ModelAndView addUserForm() {
+
+		ModelAndView model = new ModelAndView("addUser.html");
+		return model;
+	}
+
+	@RequestMapping(value = "/submitUserForm",method = RequestMethod.POST, produces = {MediaType.APPLICATION_JSON_VALUE})
+	public ResponseEntity<UserDTO> submitUserForm(@RequestBody UserDTO userDTO) {
+		System.out.println("UserMail "+userDTO.getEmail());
```
Comment (ViacheslavKruhlov) [inline src/main/java/ua/cv/tim/controller/UserController.java:62]: try don't use System.out.println
use Logger instead

File: src/main/java/ua/cv/tim/service/UserService.java [hunk 2]
```diff
@@ -26,7 +27,11 @@
 public interface UserService {
 
 	User getUserByUsername(String username);
+
+	void add(UserDTO userDTO);
+
 	void add(User user);
```
Comment (VyacheslavKruhlov) [inline src/main/java/ua/cv/tim/service/UserService.java]: method is never used

Comment (ViacheslavKruhlov) [inline src/main/java/ua/cv/tim/service/UserService.java:33]: this method is never used

File: src/main/java/ua/cv/tim/controller/UserController.java (hunk 1 above)
Comment (maximenkom) [reply src/main/java/ua/cv/tim/controller/UserController.java:62]: got it

**Pull request merged**